JoinColony/colonyCDapp
# Feat/15967008 staged flow

## Description

- Added flow for staged payment using permission and reputation method
- Issue - https://github.com/JoinColony/colonyCDapp/issues/2669

## Testing

* Install Staged payment extension and give this extension owner permission
* Create staged payment action and go to payment step (every step before payment should work the same as in payment builder action)
